### ISSUE TITLE
NTR: Fumarylacetoacetate hydrolase activity low in fibroblast (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8220]
New term request Fumarylacetoacetate hydrolase activity low in fibroblast (elthat_110828180231): FAH  UMLS: 1848698

**Comment by pnrobinson** [comment]
@MickeySegal
activity can be measured in multiple tissues, see https://www.nature.com/articles/pr1983130.pdf

Adding new term: 
- ID: HP:6000827
- label: Diminished tissue fumarylacetoacetate hydrolase activity
- parents: [Abnormal tissue enzyme concentration or activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: 
- def: Concentration or activity of fumarylacetoacetate hydrolase (FAH; EC 3.7.1.2) below the lower limit of normal. This enzyme can be measured in multiple tissues including leukocytes and cultured fibroblasts.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8220
- synonyms:
